src/components/SelectionList/UserListItem.js

```diff
@@ -7,7 +7,7 @@ import Tooltip from '@components/Tooltip';
 import useThemeStyles from '@styles/useThemeStyles';
 import {userListItemPropTypes} from './selectionListPropTypes';
 
-function UserListItem({item, isFocused = false, showTooltip}) {
+function UserListItem({item, textStyle, alternateTextStyle, showTooltip}) {
```

Comment:
Rename the props to  `textStyles`, `alternateTextStyles` and add them to `commonListItemPropTypes`